<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CellDetection</class>
 <widget class="QMainWindow" name="CellDetection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGraphicsView" name="TimeSeriesGraph"/>
      </item>
      <item>
       <widget class="QTableWidget" name="ContoursTable"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="ContoursCheckBox">
        <property name="text">
         <string>Show All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="LabelsCheckBox">
        <property name="text">
         <string>Labels</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>308</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
